src/mount/extensions/autostart.clj

```diff
@@ -0,0 +1,49 @@
+(ns mount.extensions.autostart
```

Comment:
Could you add a small description to the namespace, for documentation purposes?

Reply:
Added.